### ISSUE TITLE
the problem for training loss

### ISSUE THREAD

**Issue by slz929** [valeoai/MTAF#3]
I train the Mdis method for one source and one target, but I am confused for the loss, and I plot by tensorboard.
And as I think, the adv loss should walk higher and the discrimitor loss should walk low.
but in the loss below, the two losses oscillate around a number, 

**whats wrong with it?**

And my training config:
adv loss weight: 0.5 
adv learning rate: 1e-5 
seg learning rate: 1.25e-5

adversarial loss of one source and one target  
![image](https://user-images.githubusercontent.com/17669514/146863115-ebd6fe1f-ab0f-45d7-abb8-8efc65cdc936.png)

dicriminator loss of one source and one target
![image](https://user-images.githubusercontent.com/17669514/146863204-b966ea44-c079-49e9-b9b0-9f11f2217caf.png)

**Comment by SportaXD** [comment]
What you observe is a standard behavior of adversarial training. Since the adversarial loss and the discriminator loss have opposite objectives, you shouldn't observe one or the other going down to 0 significantly. Observing this kind of behavior shows that your training is well balanced between discriminator training and adversarial training. Overall, you shouldn't bother too much with these loss values.

On another note, you shouldn't use Mdis for training on one source and one target but a standard AdvEnt for instance. Multi-Dis. is designed to handle more than one target.

Hope it helps.

**Comment by slz929** [comment]
@SportaXD 
Thanks for quick reply.
I am clear. 
I just fix the train_mdis fucntion to train with 1source 1target to generate the loss plot above. And in coincidence, it is same with the train_advEnt function to update the loss and etc.
